event-triggers.md

```diff
@@ -91,7 +91,7 @@ Transform and load data into external data-stores. Check out this demo and tutor
 * [Try it out](https://shahidh.in/hasura-serverless-etl)
 * [Tutorial](community/examples/serverless-etl)
 
-### Building reactive UX for your async backend with realtime GraphQL
+### Building reactive UX for your async backend with real-time GraphQL
```

Comment:
```suggestion
### Building reactive UX for your async backend with realtime GraphQL
```
We are comfortable using realtime everywhere, including the [website](https://hasura.io)/